src/zkevm_specs/evm/instruction.py

```diff
@@ -290,6 +290,15 @@ def compare(self, lhs: Expression, rhs: Expression, n_bytes: int) -> Tuple[FQ, F
         assert rhs.expr().n < 256**n_bytes, f"rhs {rhs} exceeds the range of {n_bytes} bytes"
         return FQ(lhs.expr().n < rhs.expr().n), FQ(lhs.expr().n == rhs.expr().n)
 
+    def compare_word(self, lhs: RLC, rhs: RLC) -> Tuple[FQ, FQ]:
+        assert len(lhs.le_bytes) == 32, "Expected word to contain 32 bytes"
+        assert len(rhs.le_bytes) == 32, "Expected word to contain 32 bytes"
+        lhs_hi, lhs_lo = self.word_to_lo_hi(lhs)
+        rhs_hi, rhs_lo = self.word_to_lo_hi(lhs)
```

Comment:
right hand side here?
```suggestion
        rhs_hi, rhs_lo = self.word_to_lo_hi(rhs)
```

Reply:
fixed